### PR TITLE
Kindly Review

### PR DESCRIPTION
**Pull request by ZehaIrawan**

### CODE REVIEW

File: spec/enumerable_methods_spec.rb [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      expect(arr.my_none? { |i| i >= 5 }).to eq(false)
+      expect(arr.my_none? { |i| i > 8 }).to eq(true)
+      expect(arr.my_none? { |i| i }).to eq(true)
+      expect(arr.my_none? { |i| i == 0 }).to eq(true)
+      expect(arr.my_none? { |i| i < 2 }).to eq(false)
+    end
+  end
+end
+
+RSpec.describe Enumerable do
```
Comment (inhaq) [inline spec/enumerable_methods_spec.rb]: No need to repeat this line, you can encapsulate all the tests within first one

**Review by inhaq — CHANGES_REQUESTED** [review]
### Hi team 👋🏻
Good job on writing tests with good practices. Although you're repeating one line for each function which i mentioned, remove that line and also update your variables to make it work.
Once fixed, kindly submit for review.

File: spec/enumerable_methods_spec.rb (hunk 1 above)
Comment (JangkarBumi) [reply spec/enumerable_methods_spec.rb]: We tried using that approach first, the rubocop will complain about too many lines, that's why we divide it

**Review by inhaq — APPROVED** [review]
### Well done Team 👏🏻
Tests are following good practices 🏆. Kindly update readme before merging.
It's 👍🏻, go ahead and submit a project completion form.
Good luck with the next projects 🎉

**Pull request merged**